.meteor/packages

```diff
@@ -180,3 +180,4 @@ assistify:help-request
 assistify:ai
 assistify:defaults
 chatpal:search
+monbro:mongodb-mapreduce-aggregation
```

Comment:
I don't feel it's necessary to use this package. The mongo `rawCollection` should do the trick.
The metorhack-package you mentioned [does exactly that](https://github.com/meteorhacks/meteor-aggregate/blob/master/index.js) - and is much smaller (20LOC). Any reason not to use that one?

Reply:
I tried to use the metorhack-package but it didn't work.